helixbass/line-runner
# Run line every measure

In this PR:
- listen to external MIDI clock
- fire a hardcoded run of sixteenth notes at the beginning of every bar in time with the MIDI clock

To test:
In Ableton, to send the MIDI clock to the program you (a) start the program running at which point it should be visible as both a virtual MIDI input + output (b) go to Preferences -> Link Tempo MIDI and under MIDI Ports you should see "Out: Line runner" and check the Sync box. If you then set a MIDI track's MIDI input to be "Line runner" (and assign a MIDI instrument) and start the clock (by clicking the play button) you should hear a line repeating every measure
